hack/generate-latest-csv.sh

```diff
@@ -27,5 +28,8 @@ echo -e "\tNOOBAA_CORE_IMAGE=$NOOBAA_CORE_IMAGE"
 echo -e "\tNOOBAA_DB_IMAGE=$NOOBAA_DB_IMAGE"
 echo -e "\tOCS_IMAGE=$OCS_IMAGE"
 echo -e "\tOCS_MUST_GATHER_IMAGE=$OCS_MUST_GATHER_IMAGE"
+echo -e "\tVOLUME_REPLICATION_OPERATOR_IMAGE=$VOLUME_REPLICATION_OPERATOR_IMAGE"
```

Comment:
`VOLUME_REPLICATION_OPERATOR_IMAGE` should not be part of this commit? Maybe a 2nd commit, or an other PR?

Reply:
> `VOLUME_REPLICATION_OPERATOR_IMAGE` should not be part of this commit? Maybe a 2nd commit, or an other PR?

its just a harmless echo statement, I guess its fine.

Reply:
It is not relevant what it is, harmless or not. It is unrelated to this PR, so it should not be included.

Reply:
Obviously, the same counts for `ROOK_CSIADDONS_IMAGE` and any other non-NFS related modification.

Reply:
done, added another commit, PTAL @nixpanic